lib/components/programs-list/index.tsx

```diff
@@ -19,42 +29,68 @@ const ProgramListing: React.FC<ProgramListingProps> = ({
   recordingLink,
 }) => {
   const breakpoint = useBreakpoint();
+  const editProgramModalManagement = useDisclosure();
   let useMobileLayout = breakpoint && PAGE_SIZES.includes(breakpoint);
 
   return (
-    <Flex flexDirection={useMobileLayout ? 'column' : 'row'} gap={4}>
-      <Flex flexDirection="column" alignItems={useMobileLayout ? 'start' : 'end'} minWidth={'40%'}>
-        <Heading textAlign={useMobileLayout ? 'left' : 'right'}>{date}</Heading>
-        <Text>{time}</Text>
-      </Flex>
-      <Box
-        h={useMobileLayout ? '4px' : 'maxContent'}
-        minWidth={useMobileLayout ? 'maxContent' : '4px'}
-        bg={'var(--magenta)'}
-      />
-      <Flex flexDirection="column" alignItems="start">
-        <Heading>{title}</Heading>
-        <Text fontSize="lg" whiteSpace="pre-line">
-          {description}
-        </Text>
-        <Link
-          href={recordingLink || registerLink}
-          isExternal
-          _hover={{ textDecoration: 'none' }}
-          _focus={{ outline: 'none' }}>
+    <>
+      <EditProgramModal modalManagement={editProgramModalManagement} />
```

Comment:
Small thing but I want to know your thoughts: In the Figma designs, we don't have a version with both buttons yet since the Admin version technically can't see the button

<details>
<summary>Figma Comparison</summary>
<br>
![image](https://user-images.githubusercontent.com/71473137/200030680-d4fe71c8-fae8-45e2-adc7-c00e9a94b669.png)
</details>

Seems like it would make sense to keep the Register here button on the Admin version
Should we distance these buttons or add an "edit" icon to make them look more distinct?